data/themes/darktable.css

```diff
@@ -483,7 +456,7 @@ entry selection
   padding: 0 2pt 2pt 0.5em;
   font-weight: normal;
   font-family: sans-serif;
-  font-size: larger;
+  font-size: 1.1em;
```

Comment:
This one is already set on master like that. Not a changed one, probably about the fact I started this on next branch.